package.json

```diff
@@ -106,6 +106,11 @@
           "description": "String used for indentation",
           "default": "  "
         },
+        "todo.automaticSymbol": {
```

Comment:
I'm not sure about this name, maybe `todo.symbols.auto` could be more in line with the other settings?